### ISSUE TITLE
Build with ruby support

### ISSUE THREAD

**Issue by avluis** [dinoex/iroffer-dinoex#8]
I've been attempting to create a Docker image with ruby support but so far I've only gotten 3.30 to enable while the snapshot will refuse no matter the libruby version.

Got any tips?

---
As soon as I write this, I happen to land on the right combination:
Debian Wheezy with ruby1.8-dev, ruby1.8 and libruby1.8

**Comment by dinoex** [comment]
The Documentation was already updated to match Debian Jessie.

**Comment by avluis** [comment]
That is correct, but unfortunately I haven't been able to compile with ruby support following said documentation, which was the reason for my question.
Thanks for responding though.

**Comment by dinoex** [comment]
in older Debian Releases "libruby-extras" where needed:

https://github.com/dinoex/iroffer-dinoex/commit/7246841f815268d8bc317ea99a726793a2a1d2ed#diff-8437f88a8e07fb1f64c693f6f155a701

**Comment by avluis** [comment]
Thank you.
I got it working after taking a look at the install logs.

For jessie, I needed:
```ruby
ca-certificates
gcc
libc-dev
libcurl4-openssl-dev
libgeoip-dev
libminiupnpc-dev
libssl-dev
make
ruby
ruby-dev
```

Since I was creating this for a Docker container, once iroffer was built, all I needed to have installed was:
```
libcurl3
libgeoip1
libminiupnpc10
ruby
```

Thanks for the heads up in regards to `libruby-extras` -- I was attempting to install that at the time but eventually found the new ruby dependencies for Debian stable (jessie).
